### PR TITLE
[zh] translation for admin2

### PR DESCRIPTION
**Pull request by steven-my**
related: #29329
Translation for the 'admin 2' section in issue 29329.

### CODE REVIEW

File: content/zh/docs/tasks/administer-cluster/network-policy-provider/antrea-network-policy.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+weight: 10
+---
+
+<!-- overview -->
+<!--
+This page shows how to install and use Antrea CNI plugin on Kubernetes.
+For background on Project Antrea, read the [Introduction to Antrea](https://antrea.io/docs/).
+-->
+本页展示了如何在 kubernetes 中安装和使用 Antrea CNI 插件。
+了解 Antrea 项目的背景，阅读 [Antrea 介绍](https://antrea.io/docs/)
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/network-policy-provider/antrea-network-policy.md:20]:
```suggestion
要了解 Antrea 项目的背景，请阅读 [Antrea 介绍](https://antrea.io/docs/)。
```

File: content/zh/docs/tasks/administer-cluster/network-policy-provider/antrea-network-policy.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+## {{% heading "prerequisites" %}}
+
+<!--
+You need to have a Kubernetes cluster. Follow the
+[kubeadm getting started guide](/docs/reference/setup-tools/kubeadm/) to bootstrap one.
+-->
+你需要拥有一个 kuernetes 集群。
+遵循 [kubeadm 入门指南](/docs/reference/setup-tools/kubeadm/) 自行创建一个。
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/network-policy-provider/antrea-network-policy.md:29]:
```suggestion
遵循 [kubeadm 入门指南](/zh/docs/reference/setup-tools/kubeadm/)自行创建一个。
```

File: content/zh/docs/tasks/administer-cluster/network-policy-provider/antrea-network-policy.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+<!-- steps -->
+
+<!--
+## Deploying Antrea with kubeadm
+
+Follow [Getting Started](https://github.com/vmware-tanzu/antrea/blob/main/docs/getting-started.md) guide to deploy Antrea for kubeadm.
+-->
+## 使用 kubeadm 部署 Antrea
+遵循 [入门](https://github.com/vmware-tanzu/antrea/blob/main/docs/getting-started.md) 指南
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/network-policy-provider/antrea-network-policy.md:39]:
```suggestion
遵循[入门](https://github.com/vmware-tanzu/antrea/blob/main/docs/getting-started.md)指南
```

File: content/zh/docs/tasks/administer-cluster/network-policy-provider/antrea-network-policy.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+为 kubeadm 部署 Antrea 。
+
+## {{% heading "whatsnext" %}}
+
+<!--
+Once your cluster is running, you can follow the [Declare Network Policy](/docs/tasks/administer-cluster/declare-network-policy/) to try out Kubernetes NetworkPolicy.
+-->
+一旦你的集群已经运行，你可以遵循 
+[声明网络策略](/zh/docs/tasks/administer-cluster/declare-network-policy/)
+来尝试 Kubernetes NetworkPolicy
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/network-policy-provider/antrea-network-policy.md:49]:
```suggestion
来尝试 Kubernetes NetworkPolicy。
```

File: content/zh/docs/tasks/administer-cluster/network-policy-provider/cilium-network-policy.md [hunk 5]
```diff
@@ -33,62 +33,71 @@ For background on Cilium, read the [Introduction to Cilium](https://docs.cilium.
...
 
 <!--
-Mount the BPF filesystem:
+For minikube you can install Cilium using its CLI tool. Cilium will
+automatically detect the cluster configuration and will install the appropriate
+components for a successful installation:
 -->
-挂载 BPF 文件系统：
+对于 minikube 你可以使用 Cilium 的 CLI 工具安装它。
+Cilium 将自动检测集群配置并为一次成功的安装选择合适的组件。
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/network-policy-provider/cilium-network-policy.md:67]:
```suggestion
Cilium 将自动检测集群配置并为成功的集群部署选择合适的组件。
```

File: content/zh/docs/tasks/administer-cluster/reserve-compute-resources.md [hunk 6]
```diff
@@ -333,14 +333,14 @@ respectively.
...
-调度器将 `Allocatable` 视为 Pod 可用的 `capacity`（资源容量）。
+调度器将 'Allocatable' 视为 Pod 可用的 `capacity`（资源容量）。
 
-`kubelet` 默认对 Pod 执行 `Allocatable` 约束。
-无论何时，如果所有 Pod 的总用量超过了 `Allocatable`，驱逐 Pod 的措施将被执行。
+`kubelet` 默认对 Pod 执行 'Allocatable' 约束。
+无论何时，如果所有 Pod 的总用量超过了 'Allocatable'，驱逐 Pod 的措施将被执行。
 有关驱逐策略的更多细节可以在
-[这里](/zh/docs/tasks/administer-cluster/out-of-resource/#eviction-policy)找到。
+[节点压力驱逐](/zh/docs/concepts/scheduling-eviction/node-pressure-eviction/) 页找到。
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/reserve-compute-resources.md:343]:
```suggestion
[节点压力驱逐](/zh/docs/concepts/scheduling-eviction/node-pressure-eviction/)页找到。
```

File: content/zh/docs/tasks/administer-cluster/network-policy-provider/antrea-network-policy.md (hunk 1 above)
Comment (steven-my) [reply content/zh/docs/tasks/administer-cluster/network-policy-provider/antrea-network-policy.md]: 已修改

File: content/zh/docs/tasks/administer-cluster/network-policy-provider/antrea-network-policy.md (hunk 2 above)
Comment (steven-my) [reply content/zh/docs/tasks/administer-cluster/network-policy-provider/antrea-network-policy.md]: 已修改

File: content/zh/docs/tasks/administer-cluster/network-policy-provider/antrea-network-policy.md (hunk 3 above)
Comment (steven-my) [reply content/zh/docs/tasks/administer-cluster/network-policy-provider/antrea-network-policy.md]: 已修改

File: content/zh/docs/tasks/administer-cluster/network-policy-provider/antrea-network-policy.md (hunk 4 above)
Comment (steven-my) [reply content/zh/docs/tasks/administer-cluster/network-policy-provider/antrea-network-policy.md]: 已修改

File: content/zh/docs/tasks/administer-cluster/network-policy-provider/cilium-network-policy.md (hunk 5 above)
Comment (steven-my) [reply content/zh/docs/tasks/administer-cluster/network-policy-provider/cilium-network-policy.md]: 已修改

File: content/zh/docs/tasks/administer-cluster/reserve-compute-resources.md (hunk 6 above)
Comment (steven-my) [reply content/zh/docs/tasks/administer-cluster/reserve-compute-resources.md]: 已修改

**Review by tanjunchen — COMMENTED** [review]
/lgtm

**Pull request merged**